spec.html

```diff
@@ -28464,78 +28323,47 @@ <h1>Math.cosh ( _x_ )</h1>
-        <ul>
-          <li>
-            If _x_ is *NaN*, the result is *NaN*.
-          </li>
-          <li>
-            If _x_ is *+0*, the result is *+0*.
-          </li>
-          <li>
-            If _x_ is *-0*, the result is *-0*.
-          </li>
-          <li>
-            If _x_ is *+&infin;*, the result is *+&infin;*.
-          </li>
-          <li>
-            If _x_ is *-&infin;*, the result is -1.
-          </li>
-        </ul>
+        <p>When the `Math.expm1` method is called with one argument _x_, the following steps are taken:</p>
+        <emu-alg>
+          1. Let _n_ be ? ToNumber(_x_).
+          1. If _n_ is *NaN*, _n_ is *-0*, _n_ is *+0*, or _n_ is *+&infin;*, return _n_.
+          1. If _n_ is *-&infin;*, return -1.
+          1. Return an implementation-approximated value representing the result of subtracting 1 from the exponential function of _n_.
+        </emu-alg>
       </emu-clause>
 
       <emu-clause id="sec-math.floor">
         <h1>Math.floor ( _x_ )</h1>
-        <p>Returns the greatest (closest to *+&infin;*) Number value that is not greater than _x_ and is an integer. If _x_ is already an integer, the result is _x_.</p>
-        <ul>
-          <li>
-            If _x_ is *NaN*, the result is *NaN*.
-          </li>
-          <li>
-            If _x_ is *+0*, the result is *+0*.
-          </li>
-          <li>
-            If _x_ is *-0*, the result is *-0*.
-          </li>
-          <li>
-            If _x_ is *+&infin;*, the result is *+&infin;*.
-          </li>
-          <li>
-            If _x_ is *-&infin;*, the result is *-&infin;*.
-          </li>
-          <li>
-            If _x_ is greater than 0 but less than 1, the result is *+0*.
-          </li>
-        </ul>
+        <p>Returns the greatest (closest to *+&infin;*) integral Number value that is not greater than _x_. If _x_ is already an integral Number, the result is _x_.</p>
+        <p>When the `Math.floor` method is called with one argument _x_, the following steps are taken:</p>
+        <emu-alg>
+          1. Let _n_ be ? ToNumber(_x_).
+          1. If _n_ is *NaN*, _n_ is *+0*, _n_ is *-0*, _n_ is *+&infin;*, or _n_ is *-&infin;*, return _n_.
+          1. If _n_ &gt; 0 and _n_ &lt; 1, return *+0*.
+          1. If _n_ is an integral Number, return _n_.
+          1. Return the greatest (closest to *+&infin;*) integral Number value that is not greater than _n_.
+        </emu-alg>
         <emu-note>
           <p>The value of `Math.floor(x)` is the same as the value of `-Math.ceil(-x)`.</p>
         </emu-note>
       </emu-clause>
 
       <emu-clause id="sec-math.fround">
         <h1>Math.fround ( _x_ )</h1>
-        <p>When `Math.fround` is called with argument _x_, the following steps are taken:</p>
+        <p>When the `Math.fround` method is called with one argument _x_, the following steps are taken:</p>
```

Comment:
```suggestion
        <p>When the `Math.fround` method is called with one argument _x_, the following steps are taken:</p>
```
or perhaps, remove "one" from all these preambles? "is called with one" appears 36 times in the spec, and "is called with arg" appears 113 times, so it seems like it'd be more consistent to go with the more common form and make them all "is called with argument".

(Additionally, by specifying the number of arguments there's a mild implication that if it's called with zero or 2+ args, different steps are taken, which isn't the case)

Reply:
Yes, I would expect and prefer that all these explicit mentions of a count followed by a short list where the reader can easily count for themselves should remove the count :-)